newton/solvers/mujoco/solver_mujoco.py

```
@@ -2009,6 +2038,97 @@
+                    pos=site_pos,
+                    quat=quat_to_mjc(site_quat),
+                )
+
+        # -----------------------
+        # add tendons to MuJoCo model
+        
+        if hasattr(model, 'tendon_name') and model.tendon_count > 0:
+            for tendon_idx in range(model.tendon_count):
+                tendon_name = model.tendon_name[tendon_idx]
+                tendon_type = model.tendon_type[tendon_idx]
+                
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)